### PR TITLE
Bump ruby versions

### PR DESCRIPTION
**Pull request by alsutton**
Update the ruby versions to track the latest releases so testing covers all basis.

### CODE REVIEW

**Review by shs96c — COMMENTED** [review]
You need to run "bundle install" and ensure that the Docker images are also updated.

Edit: Apologies. I thought this was OW. Will review properly :)

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 language: ruby
 cache: bundler
 rvm:
-  - 2.2.7
-  - 2.3.4
-  - 2.4.1
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
```
Comment (shs96c) [inline .travis.yml:7]: My impression was that we were waiting until 2.5.1 was out before we looked at 2.5....

**Comment by alsutton** [comment]
Some projects are already running with 2.5.0

File: .travis.yml (hunk 1 above)
Comment (alsutton) [reply .travis.yml:7]: Hopper is already on 2.5.0

**Comment by alsutton** [comment]
Anyone fancy giving this a review?

**Review by marcinwyszynski — APPROVED** [review]

**Pull request merged**